src/logmessage.cpp

```diff
@@ -27,35 +27,35 @@ namespace g3 {
    // helper for setting the normal log details in an entry
    std::string LogDetailsToString(const LogMessage& msg) {
       std::string out;
-      out.append("\n" + msg.timestamp() + "\t"
```

Comment:
lol. I couldn't care less if the newline is first or last since the end result is the same. I.e this is fine with me. The same goes for the removal of the `"` quotation marks around the log entry. They are not needed of course. 

Since this might affect other users I will call on the to voice their opinion.  They can diff these and voice any objections:
@jkhoogland , @cstamatopoulos , @ctapmex , @AndreasSchoenle 
[original-look.txt](https://github.com/KjellKod/g3log/files/668044/original-look.txt)
[new-look.txt](https://github.com/KjellKod/g3log/files/668045/new-look.txt)